README.md

```diff
@@ -1 +1,144 @@
-# decode-morse-code
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Morse Decoder <a name="about-project"></a>
+
+**Morse Decoder** A Morse Decoder is used to decode the Morse code to an actual readable word or message.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby.rb</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Linters</summary>
+  <ul>
+    <li><a href="https://github.com/microverseinc/linters-config/tree/master/ruby">Linters</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Relational Database**
+- **Uses Synthetic Primary Key**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Undercostruction](<replace-with-your-deployment-URL>)
```

Comment:
- [ ] Good job so far, kindly remove the live demo section as you dont have a deployed link for this project, you can add the section back when you deploy currently the link you added shows a 404 error page 👍🏿